src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
You should be able to move title/subtitle strings with `position: relative` and (negative?) `top` values. Though, could you go over this layout with someone who is more familiar with CSS than us? We are currently floating left a bunch of elements and we want to have these two text rows span the logo:
┌─── ┬────┐
│ logo ├────┤
└─── ┴────┘